pkg/client/typed/dynamic/client.go

```diff
@@ -112,6 +113,17 @@ func (rc *ResourceClient) List(opts v1.ListOptions) (*runtime.UnstructuredList,
 	return result, err
 }
 
+// UnversionedList returns a list of objects for this resource.
+func (rc *ResourceClient) UnversionedList(opts api.ListOptions) (*runtime.UnstructuredList, error) {
```

Comment:
Yeah, please don't add this function-- make the regular list take a runtime.Object for the ListOptions. @krousey